tests/test_initiate.py

```diff
@@ -3,5 +3,4 @@
 
```

Comment:
Correct placeholder text for library description, pros features and code example in the documentation index page.